src/bot_commands.py

```diff
@@ -610,7 +629,8 @@ def _authorize(
         account: Union[AccountId, str],
```

Comment:
Why? Why? What's the point here? The whole philosophy for this refactor was that the server would nto be directly called by the UI file (this one)

Reply:
Along with that I totally reject the exporting of `_get_account(Union[AccountId, str], Server) -> Account`. It's in that file for a reason.

Reply:
apologies I had completely misread that code I started on this late at night and am only realizing what was originally going on yeah that is wrong

Reply:
nice good late night coding after which you will definitely look back and wonder "what the hell"

as it should be